mlflow/openai/__init__.py

```diff
@@ -429,6 +470,23 @@ def _has_content_and_role(d):
     return "content" in d and "role" in d
 
 
+class _Message:
+    def __init__(self, message):
+        self.raw_content = message.get("content")
+        self.role = message.get("role")
+        self.variables = _parse_format_fields(self.raw_content)
+
+    def format(self, **params):
+        if missing_params := set(self.variables) - set(params):
+            raise mlflow.MlflowException.invalid_parameter_value(
+                f"Missing parameters: {missing_params}"
```

Comment:
Can this message be a little more verbose to let the users know the structure that is expected?

Reply:
got it, I'll update the message :)